lib/cfg-tree.c

```diff
@@ -897,6 +897,12 @@ cfg_tree_propagate_expr_node_properties_to_pipe(LogExprNode *node, LogPipe *pipe
     pipe->expr_node = node;
 }
 
+static gboolean
+_is_log_path_name_unique(CfgTree *self, const LogExprNode *node)
+{
+  return g_hash_table_insert(self->log_path_names, g_strdup(node->name), NULL);
```

Comment:
> Starting from GLib 2.40, this function returns a boolean value
2.32 is our min version currently :(

Reply:
Wow, what a nice catch, thanks! :)

I have added a `contains()` call before it, which is available from 2.32: https://developer-old.gnome.org/glib/2.32/glib-Hash-Tables.html#g-hash-table-contains

Reply:
Thanks.